apis/events/protobuf/v1/event.proto

```diff
@@ -41,9 +41,10 @@ enum EventAction {
   SwitchOver = 6;
   AddChild = 7;
   RemoveChild = 8;
-  NvmePathSuspect = 9;
-  NvmePathFail = 10;
-  NvmePathFix = 11;
+  OnlineChild = 9;
```

Comment:
Might be worth clarifying if this is RPC of childState change, maybe like so:
```suggestion
  AddChildRpc = 7;
  RemoveChildRpc = 8;
  OnlineChildRpc = 9;
```
Wdyt?

Reply:
I take it back, as per name `OnlineChild` suggest it's an action, rather than `ChildOnline` or `ChildIsOnline`.

Reply:
Ok. Yes that would be better.

Actually the add child, remove child, online child events are only being generated if new replicas are added to a nexus according to the current events generation. Should this be okay?